### DISCUSSION TITLE
Pull Request

### DISCUSSION THREAD

**Discussion by Bharti-Shar** [Bharti-Shar/College-Practice#7] [Q&A] [answered]
How can you create a pull request?

**Comment by Yashasvi-30** [comment, marked as answer]
# Clone the repository
git clone https://github.com/your-username/repository-name.git

# Navigate to the repository directory
cd repository-name

# Create a new branch
git checkout -b your-branch-name

# Stage changes
git add .

# Commit changes
git commit -m "Description of the changes you made"

# Push changes to GitHub
git push origin your-branch-name
